### PR TITLE
add new 0x on Scroll guide

### PR DESCRIPTION
**Pull request by jlin27**
## Context

As part of the upcoming 0x launch on Scroll, we want to encourage builders to build with 0x Swap API on Scroll with this getting started guide.

## Test

```
npm install
npm run build
npm run start
```
Review the guide at `Content > Get Started with 0x Swap API on Scroll`

### CODE REVIEW

**Comment by emilyJLin95** [comment]
@jlin27 right now it looks like you just added two screenshots - did you forget to commit some files?

**Comment by jlin27** [comment]
Can you check the the “files changed” tab?
The main file changed was a new 0x-on-scroll inside the content folder

image

File: src/contents/0x-on-scroll.mdx [hunk 1]
```diff
@@ -0,0 +1,509 @@
+---
+name: "Get Started with 0x Swap API on Scroll"
```
Comment (emilyJLin95) [inline src/contents/0x-on-scroll.mdx:2]: I think I would use the title and introduction to lead with the outcome of what you're learning

i.e., "How to fetch live swap prices" vs. being product driven first with "learn how to use 0x"

Comment (jlin27) [reply src/contents/0x-on-scroll.mdx]: makes sense. updated!

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Comment by emilyJLin95** [comment]
@PrintRH 
Seeing this for Merging is blocked
`Missing successful active Preview deployment.`

Not sure if I have permissions to start a new one?

**Pull request merged**